BitwardenShared/Core/Platform/Services/KeyConnectorService.swift

```diff
@@ -139,7 +139,7 @@ extension DefaultKeyConnectorService: KeyConnectorService {
 
     func getManagingOrganization() async throws -> Organization? {
         try await organizationService.fetchAllOrganizations()
-            .first { $0.useKeyConnector && !$0.isAdmin }
+            .first { $0.keyConnectorEnabled && !$0.isAdmin }
```

Comment:
🤔 By checking in the clients solution I see they also have an additional check for `isProviderUser` as you can see here:
https://github.com/bitwarden/clients/blob/887d1924a331a292c5425136aac39b14166cf814/libs/common/src/auth/services/key-connector.service.ts#L125
I'll ask around to see if we'd need to add that here as well.